test/steps/api/3_then.js

```diff
@@ -66,7 +66,7 @@ export function itShouldBroadcastTheTransaction() {
 
 export function itShouldResolveToTheAPIResponse() {
 	const { returnValue, apiResponse } = this.test.ctx;
-	return returnValue.should.be.fulfilledWith(apiResponse);
+	return returnValue.should.be.eventually.eql(apiResponse);
```

Comment:
`.should.eventually.eql`

Reply:
I think we need `be` verb in natural language?